core/types/block.go

```diff
@@ -99,6 +99,9 @@ type Header struct {
 
 	// ExcessBlobGas was added by EIP-4844 and is ignored in legacy headers.
 	ExcessBlobGas *uint64 `json:"excessBlobGas" rlp:"optional"`
+
+	// ParentBeaconRoot was added by EIP-4788 and is ignored in legacy headers.
+	ParentBeaconRoot *common.Hash `json:"parentBeaconBlockRoot" rlp:"optional"`
```

Comment:
Yeah I was thinking it could be used for something like that. We can delete it until the next fork so I suggest we leave it for now, even if its empty. We did omit withdrawals but I'm not sure that was the best idea for compatibility.